Assets/Scripts/Rendering/QueueManager.cs

```diff
@@ -160,7 +161,8 @@ of queues.
             Vector3 position = new Vector3(sXZ * (i % dimensions[queueType][0]), 0, sXZ * (i / dimensions[queueType][0]));
             Vector3 queueManagerHeight = new Vector3(0, sY*2, 0);
 
-            GameObject queueGameObject = new GameObject(queue.queueName, typeof(Queue));
+            string uniqueQueueName = qmName + "." + queue.queueName;
```

Comment:
I suggest a small change as a good coding practice - create a constant variable `QM_NAME_DELIMITER = "."` or whatever and then have this as `qmName + QM_NAME_DELIMITER + queue.queueName;` Similarly with other occurrences. It is cleaner for future operability because you just need to change the constant string at one place and also it's easier to understand

Reply:
good idea